static/app/views/performance/newTraceDetails/traceTypeWarnings/errorsOnlyWarnings.tsx

```diff
@@ -83,7 +83,7 @@ function PerformanceSetupBanner({
     return (
       <Alert type="info" showIcon>
         {tct(
-          "Some of the projects associated with this trace don't support performance monitoring. To learn more about how to setup performance monitoring, visit our [documentation].",
+          "Some of the projects associated with this trace don't support tracing. To learn more about how to setup tracing, visit our [documentation].",
```

Comment:
I am not sure if the 1:1 translation works here: What do others think of rewording the whole sentence?
```suggestion
          "Some of the projects associated with this trace don't support "Tracing". To learn more about how to setup tracing, visit our [documentation].",
```

or adding Sentry:
```suggestion
          "Some of the projects associated with this trace don't support Sentry tracing. To learn more about how to setup tracing, visit our [documentation].",
```

Reply:
agree that the 1:1 translation sounds weird. I would go with your first suggestion.